pkg/parsers/filters/parse.go

```diff
@@ -36,9 +37,10 @@ func ParseFlag(arg string, prev Args) (Args, error) {
 	return filters, nil
 }
 
+// ErrorBadFormat bad format for filter
```

Comment:
@runcom Agreed. `ErrBadFormat` is the most idiomatic, unless these are typed constants for a locally defined `Error` type.